amundsen_application/static/js/components/TableDetail/OwnerEditor/index.tsx

```diff
@@ -178,10 +178,7 @@ export class OwnerEditor extends React.Component<
           <button
             className="btn btn-default add-button"
             type="submit"
-            aria-label="Add Item"
-          >
-            <span aria-hidden="true">Add</span>
-          </button>
+          > Add </button>
```

Comment:
Removed the aria-label as well as the sr-only because when selecting the edit button the sr would read edit on Owners, which means that if this option is selected it can be assumed that when the sr reads Add, it will be related to adding an Owner. Trying to have both a sr-only plus the label of the button itself would lead to confusion because i would read "add item" and also "add", before with aria-hidden it would just hide "Add" and say "add item", making this not an issue. We could also keep aria-hidden but idk if this also has support gaps.